blog/2015/01/instructor_debriefing-01-27.html

```diff
@@ -0,0 +1,127 @@
+category: ["Workshops"]
+---
+
+<!-- start excerpt -->
+<p>There have been nine new workshops since mid-January. Greg Wilson
+  and I recently hosted a second post-workshop debriefing session for
+  January to capture more experiences and lessons learned from
+  instructors in the field. This meeting was attended by 10 instructors
+  covering four of these recent workshops. In addition to the usual
+  two-day format, there was also an extended, three-day version and a
+  Data Carpentry workshop. We discussed how the workshops went, what
+  worked, what didn't, and what could be improved.</p>
+<!-- end excerpt -->
+<p>
+  There is always something new to learn from instructors' experiences,
+  particularly from this meeting, as we had the chance to discuss a Data
+  Carpentry workshop at Cornell University as well as a three-day "Beyond
+  the Code" workshop at the NERC/University of Leeds. Some highlights are
+  discussed below. Please see the <a
+				     href="https://etherpad.mozilla.org/swc-debriefing-2015-01-27">etherpad</a>
+  for detailed notes.
+</p>
+
+http://software-carpentry.org/blog/2014/12/an-advanced-short-course-in-leeds.html
+
+
+<h3>Instruction Methods and Technology</h3>
+<ul>
+  <li>Dual screens were available at the University of Washington
+    and Oxford workshops, which was reported to have worked very well.
+    We've discussed before that it is helpful for the instructor to use a
+    tablet (or second computer) to consult the lessons while live coding.
+    Having two projectors seems to be even better by making it possible
+    for the participants to to refer to the lessons material while also
+    keeping the code visible.</li>
+  <li>Some take-home lessons were that live coding on screen is much
+    better received than prepared slide decks. Also, you can't have to
+    many hands-on exercises, which are popular with participants.</li>
+  <li>Another common theme from this retrospective was that it was
+    difficult to estimate and/or stick to lesson timing, which is probably
+    related in part to new or newly refactored lesson materials</li>
+  <li>We've noticed in workshops that adoption and use of the
+    etherpad can be somewhat spotty, not gaining traction or trailing off
+    as the workshop proceeds. When the etherpad is actively used by the
+    participants, it can be a very effective teaching adjunct and resource
+    for real-time discussions, answers to common questions and helpful
+    links. Almost all participants will be unfamiliar with the etherpad at
+    the start or or may not immediately see its value but, when it works,
+    it works well and builds momentum. More active participants can be
+    encouraged to post content as well to reduce the burden on the
+    instructors and helpers.</li>
+</ul>
+
+<h3>Data Carpentry Workshop at Cornell</h3>
+<a href="http://emudrak.github.io/2015-01-15-cornell/">This workshop</a>
+was run by Erika Mudrak (CSCU), Jeramia Ory (Kings College). Emily
```

Comment:
Maybe "Jeramia Ory (Kings College). Emily" -> "Jeramia Ory (Kings College) and Emily"